### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CHADBUCK13**

### CODE REVIEW

**Review by Chr728 — CHANGES_REQUESTED** [review]
Please discuss this with ghaith , it has some errors and might be missing some steps

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
-Repository for a project completed for SOEN 363
\ No newline at end of file
+
+This project aims to analyze general sentiment towards gun control. This is done through an anlysis of tweets that involve the gun control topic. 
+
+## Steps to run similar experiment with these scripts:
+
+1. Run "TwitterAPIFetcher.py" several times to accumulate enough data to perform an adequate analysis
+2. Combine the resulting .json files to a single .json file
+3. Run "TweetNLPSentimentAnalyzer_json.py" on the .json file generated in step 2
```
Comment (janong24) [inline README.md:9]: Insert a step after
>> Using JSON Editor Online (https://jsoneditoronline.org/), grab the columns you will need

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+4. Run the "CompleteProcessWrapper.py" to clean and convert the data to INSERT queries
```
Comment (janong24) [inline README.md:10]: Insert a step after
>> Using the Schema, run the CREATE TABLE queries to create the database schema

**Review by janong24 — CHANGES_REQUESTED** [review]
Insert the steps mentioned.

**Review by janong24 — APPROVED** [review]

**Review by Chr728 — APPROVED** [review]

**Pull request merged**